federatedscope/core/configs/cfg_asyn.py

```diff
@@ -10,22 +10,34 @@ def extend_asyn_cfg(cfg):
     # ---------------------------------------------------------------------- #
     cfg.asyn = CN()
 
-    cfg.asyn.use = True
-    cfg.asyn.timeout = 0
+    cfg.asyn.use = False
+    cfg.asyn.time_budget = 0
     cfg.asyn.min_received_num = 2
     cfg.asyn.min_received_rate = -1.0
+    cfg.asyn.staleness_toleration = 0
+    cfg.asyn.staleness_discount_factor = 1.0
+    cfg.asyn.aggregator = 'goal_achieved'  # ['goal_achieved', 'time_up']
+    # 'goal_achieved': perform aggregation when the defined number of feedback
+    # has been received; 'time_up': perform aggregation when the allocated
+    # time budget has been run out
+    cfg.asyn.broadcast_manner = 'after_aggregating'  # ['after_aggregating',
+    # 'after_receiving'] 'after_aggregating': broadcast the up-to-date global
+    # model after performing federated aggregation;
+    # 'after_receiving': broadcast the up-to-date global model after receiving
+    # the model update from clients
+    cfg.asyn.overselection = False
```

Comment:
To my knowledge, overselection strategy also works for sync mode.

Reply:
Since `overselection` should be used together with `staledness_toleration` in this version, and sync+overselection can be regarded as a specific case of asynchronous: sampled_num >=aggregation goal, toleration threshold=0 and broadcast after aggregation